sites/mosul/configs/openmrs/initializer_config/personattributetypes/personattributetypes_core-demo.csv

```diff
@@ -1,4 +1,4 @@
 Uuid,Void/Retire,Name,Description,Format,Foreign,Searchable
 bc851ec1-9fb7-49da-acd7-1a61168002fe,0,Telephone Number,The telephone number for the person,java.lang.String,,,
-7de00b51-3342-4191-92e8-ca4eb4ed8085,,Unknown Patient Name,Used to flag patients that cannot be identified during the check-in process,java.lang.String,,,
 14d4f066-15f5-102d-96e4-000c29c2a5d7,,Individual Email Address,The email address for the person,java.lang.String,,,
+8b56eac7-5c76-4b9c-8c6f-1deab8d3fc47,,Unknown patient,Used to flag patients that cannot be identified during the check-in process,java.lang.String,,,
```

Comment:
Why do we have 2 files in the same repo? Shouldn't a single file work out?

Reply:
Which 2 files? Are u referring to there being core (LIME overrall) and MOSUL specific configurations